views/userManagementBadge.njk

```diff
@@ -15,120 +15,133 @@
-    </tr>
-    <tr>
-        <th>Date registered: </th>
-        <td>{{user.createdAt}}</td>
-    </tr>
-    <tr>
-        <th></th>
-        {% if user.confirmation  %}
-            <td>
-                <form id="deleteForm" action="/userManagement/deleteUser/{{user.badge}}" method="post" onsubmit="return confirm('This user will be deleted!')">
-                <button type='submit' class="btn btn-primary" name='submit' value="">Delete</button>
-                </form>
-                <br>
-            {% if user.status == "User"%}
-                <form id="ChangeStatusToManager" action="/userManagement/promoteUser/{{user.badge}}" method="post" onsubmit="return confirm('This user will be Manager!')">        
-                <button type='submit' class="btn btn-primary" name='submit' value="">Promote to Manager</button>
-                </form>
-            {% elif user.status == "Manager" %}
-                <form id="ChangeStatusToManager" action="/userManagement/demoteManager/{{user.badge}}" method="post" onsubmit="return confirm('This user will be User!')">
-                <button type='submit' class="btn btn-primary" name='submit' value="">Demote to User</button>
-                </form>
-            {% endif %}
+<div class="row">
+    <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+        <div class="table-responsive">
+            <table  class="table table-bordered">
+                <tr>
+                    <th>Name</th>
+                    <td>{{user.first}} {{user.last}}</td>
+                </tr>
+                <tr>
+                    <th>Badge ID</th>
+                    <td>{{user.badge}}</td>
+                </tr>
+                <tr>
+                    <th>Email</th>
+                    <td>{{user.email}}</td>
+                </tr>
+                <tr>
+                    <th>Phone</th>
+                    <td>{{user.phone}}</td>
+                </tr>
+                <tr>
+                    <th>Signature</th>
+                    <td>{{user.signature}}</td>
+                </tr>
+                <tr>
+                    <th>Parent/Guardian signature</th>
+                    <td>{{user.ecSignature}}</td>
+                </tr>
+                <tr>
+                    <th>Emergency contact name</th>
+                    <td>{{user.ecName}}</td>
+                </tr>
+                <tr>
+                    <th>Emergency relationship</th>
+                    <td>{{user.ecRel}}</td>
+                </tr>
+                <tr>
+                    <th>Emergency phone number</th>
+                    <td>{{user.ecPhone}}</td>
+                </tr>
+                <tr>
+                    <th>Status</th>
+                    <td>{{user.status}}</td>
+                </tr>
+                <tr>
+                    <th>Updated at: </th>
+                    <td>{{user.updatedAt}}</td>
+                </tr>
+                <tr>
+                    <th>Created at: </th>
```

Comment:
This has been renamed to "Date registered:"

Reply:
I will checked on it. For the scroll bar, it's my work around for the overloading of text when we added a new column for "Date train" - see the attachment below. I don't know if there is any other way to do it or not. 
![28640646_10209095678908889_2072474457_o](https://user-images.githubusercontent.com/15096067/36964095-6e3cd764-200a-11e8-8abc-d0cbed448a18.png)

Reply:
I have double checked all the field name and all the addition in the master branch. I also fix that horizontal scroll bar.

Reply:
@huynguyencs I think the tables look good now, I think this look is much better. But seems like the field names havent been updated, if you can update them that would be good: 
![image](https://user-images.githubusercontent.com/11580217/37002724-80387db8-207f-11e8-880f-332d20039bbc.png)

Reply:
mb I changed it but forgot to save it, I just updated it and pushed it again